### PR TITLE
RAD-143, RAD-144: RDM changes to support FPS and TVAC schemas

### PR DESCRIPTION
**Pull request by WilliamJamieson**
Resolves [RAD-143](https://jira.stsci.edu/browse/RAD-143)
Resolves [RAD-144](https://jira.stsci.edu/browse/RAD-144)

This PR is the matching changes for spacetelescope/rad#364 to add maker utils support to RDM for the new models.

**Checklist**
- [ ] Added entry in `CHANGES.rst` under the corresponding subsection
- [ ] updated relevant tests
- [ ] updated relevant documentation
- [ ] Passed romancal regression testing on Jenkins / PLWishMaster. Link: https://plwishmaster.stsci.edu:8081/job/RT/job/romancal/XXX/

### CODE REVIEW

**Comment by schlafly** [comment]
This looks reasonable to me.

**Review by fjaviersanchez — COMMENTED** [review]
Thanks a lot for all this work!! It looks good to me. It just has a couple of quantities where the units don't match, although we receive those quantities in pretty arbitrary units, but we hope to convert them to physical units in the near future. I also suggested changing the default read pattern so it matches the typical TVAC and FPS read patterns (it can make the defaults pretty heavy, though). I still have to check the yaml files in rad to see if I missed anything, but it doesn't look like it here. Thanks again.

File: src/roman_datamodels/maker_utils/_ground.py [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+    ground["led_bank1_approx_wlen"] = kwargs.get(
+        "led_bank1_approx_wlen", u.Quantity(np.zeros(6, dtype=np.uint32), unit=u.nm, dtype=np.uint32)
+    )
+    ground["led_bank2_approx_wlen"] = kwargs.get(
+        "led_bank2_approx_wlen", u.Quantity(np.zeros(6, dtype=np.uint32), unit=u.nm, dtype=np.uint32)
+    )
+    ground["srcs_pd_voltage"] = kwargs.get("srcs_pd_voltage", NONUM)
+    ground["srcs_led_flux"] = kwargs.get("srcs_led_flux", NONUM)
+    ground["wfi_mce_srcs_bank1_led_i"] = kwargs.get(
+        "wfi_mce_srcs_bank1_led_i", u.Quantity(np.zeros(6, dtype=np.uint32), unit=u.cm, dtype=np.uint32)
```
[the remark this answers is not in the document]
Comment (fjaviersanchez) [reply src/roman_datamodels/maker_utils/_ground.py:127]: done!

**Review by fjaviersanchez — APPROVED** [review]
Look great. Thank you!

**Review by PaulHuwe — APPROVED** [review]
LGTM

**Pull request merged**